pkg/routeagent_driver/handlers/kubeproxy/endpoint_handler.go

```diff
@@ -64,6 +66,13 @@ func (kp *SyncHandler) LocalEndpointCreated(endpoint *submV1.Endpoint) error {
 		}
 	}
 
+	// For now we get all the cleanups
+	if kp.localCableDriver == vxlan.CableDriverName {
+		kp.installCleanupHandlers(cableCleanup.GetVXLANCleanupHandlers())
+	} else {
+		kp.installCleanupHandlers(cableCleanup.GetCleanupHandlers())
+	}
```

Comment:
I think this should be abstracted in `cableCleanup.GetCleanupHandlers ` (pass in `kp.localCableDriver`).

I'm wondering how this is going to work when we have multiple cable drivers, ie we may have `vxlan` and `libreswan` drivers running concurrent.  Can the cleanup handlers co-exist?

Reply:
The code is refactored and cleanup handlers have been removed. So marking these comments as resolved.